src/test/components/ProfileViewerContainer.test.js

```diff
@@ -0,0 +1,60 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+import * as React from 'react';
+import { shallowWithStore } from '../fixtures/enzyme';
+
+import ProfileViewerContainer from '../../components/app/ProfileViewerContainer';
+import { changeSelectedTab } from '../../actions/app';
+import { storeWithProfile } from '../fixtures/stores';
+import { getProfileFromTextSamples } from '../fixtures/profiles/make-profile';
+
+import type { TabSlug } from '../../types/actions';
+
+describe('app/ProfileViewerContainer', function() {
+  const { profile } = getProfileFromTextSamples(`
+    A A A
+    B B B
+    C C H
+    D F I
+    E E
+  `);
+
+  const tabSlugs = [
```

Comment:
Suggestion: Can this be typed as `const tabSlugs: TabSlugs[]`? That way all the code using it will know it's not just a `string`.

Reply:
sure
actually I wanted to try something more, to make sure we test all possible slugs when we add some more, and forgot.